cmd/commands/service/command.go

```diff
@@ -28,6 +28,7 @@ import (
 	service_noop "github.com/mysteriumnetwork/node/services/noop"
 	service_openvpn "github.com/mysteriumnetwork/node/services/openvpn"
 	openvpn_service "github.com/mysteriumnetwork/node/services/openvpn/service"
+	service_wireguard "github.com/mysteriumnetwork/node/services/wireguard"
```

Comment:
There is clearly something wrong with package structure inside openvpn :( service_openvpn and openvpn_service? I know it's out of this PR scope, but please don't do it with wireguard too :)